### DISCUSSION TITLE
about robot

### DISCUSSION THREAD

**Discussion by valleyss** [trickerer/Trinity-Bots#162] [Ideas]
Would you like to support aczerocore

**Comment by trickerer** [comment]
Never heard of it. Could you provide a link?

**Reply by valleyss** [reply]
sorry， that letters I've misspelled，azerothcore-wotlk

**Reply by trickerer** [reply]
It's not impossible, but is it worth it though?
Currently I'm working on a different project; once I finish the demo, I can make a version for AC to see if people are interested at all. Also, it can hardly be made into a module.

**Reply by valleyss** [reply]
The programme you wrote is very fantastic. Before playing azorothcore, I used your trinitycore-3.3.5-with-npcbots with eluna-Trinity-WotLK to merged it and compile it, but updating the source code synchronously with two of your core has always been a headache. After azorothcore WotLK updated version 4.0. I found that the AI and task system of azorothcore are more perfect, and it supports the coexistence of eluna and other modules at the same time. So I think maybe it would be perfect if the robot supports azorothcore in the form of the module. anyway, it's so appreciated what you've done.

**Reply by yiuygah** [reply]
https://github.com/azerothcore/azerothcore-wotlk

**Reply by llucky2022** [reply]
I am full of expectation~

**Comment by trickerer** [comment]
https://github.com/trickerer/Trinity-Bots/issues/206
https://github.com/trickerer/AzerothCore-wotlk-with-NPCBots

Not a module. Also #212

**Reply by valleyss** [reply]
wow! fantastic!
